CHANGELOG.md

```diff
@@ -30,6 +30,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Changelog for this and older versions can be found on the corresponding [GitHub
 releases](https://github.com/faiface/pixel/releases).
 
-[Unreleased]: https://github.com/faiface/pixel/compare/v0.8.0...HEAD
+[Unreleased]: https://github.com/faiface/pixel/compare/v0.9.0...HEAD
+[v0.9.0]: https://github.com/faiface/pixel/compare/v0.8.0...v0.9.0
```

Comment:
By using the https://github.com/faiface/pixel/compare/v0.8.0...v0.9.0 URL here you allow users to click on the headlines in the rendered CHANGELOG.md (i.e. on GitHub) and get the _actual_ full changes that happened between versions. The same is also true to see all Unreleased (i.e. untagged) changes. Give it a try in the Preview of this file :)